packages/control-property-editor/.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Please rename this file to `.eslintrc.js`. This is what we use in all other packages and it is needed to avoid an eslint issue https://github.com/SAP/open-ux-tools/pull/1201

Reply:
renamed.